pkgs/by-name/ka/kanha/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  stdenv,
+  darwin,
+  pkgs,
```

```suggestion
  openssl,
```

never pass in `pkgs`